### PR TITLE
corrected tests in game stats spec file and added a test for the all_…

### PR DESCRIPTION
**Pull request by LancePants97**
Corrected my game_stats_spec tests and added one last test for my all_scores helper method.

### CODE REVIEW

File: spec/game_stats_spec.rb
Comment (cheeleertr) [inline spec/game_stats_spec.rb:1]: Nice quick update of the spec file

File: spec/game_stats_spec.rb [hunk 1]
```diff
@@ -76,4 +76,10 @@
         end
     end
 
+    describe '#all_scores' do
```
Comment (cheeleertr) [inline spec/game_stats_spec.rb:79]: I like how you did all the possible results

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
-require 'simplecov'
-SimpleCov.start 
+# require 'simplecov'
```
Comment (cheeleertr) [inline spec/spec_helper.rb:1]: will have to reject the commented out simplecov when we merge

Comment (JCL461437) [reply spec/spec_helper.rb:1]: Agree with Chee.

**Review by JCL461437 — APPROVED** [review]
Looks great Lance!

**Pull request merged**